community_server/src/Controller/AppRequestsController.php

```diff
@@ -265,6 +266,89 @@ public function createCoins()
         
     }
     
+    public function getBalance($session_id = 0)
+    {
+
+        $login_result = $this->requestLogin($session_id, false);
+        if($login_result !== true) {
+            return $this->returnJson($login_result);
+        }
```

Comment:
The request login function now look at first for the session_id in php session, if not found in Cookie GRADIDO_LOGIN which will be set by login-server on login, even on login by json-request and if this isn't found, then it uses session_id from caller. 
So like you have said the login will be handled by cookie as it is standard, rather than with a parameter in every request.

Reply:
So the session_id parameter is only needed is frontend is running of another domain.

Reply:
But that should be the standard, right? N communities to 1 login server. What about a Bearer Header (thats the authentication standard I believe)

Reply:
I made an Issue: https://github.com/gradido/gradido/issues/213

Reply:
We can also use Bearer Header. The only change I notice after reading about it is sending the session_id in Authorization Header. I can update my code to do that.